src/averageAges.js

```diff
@@ -15,11 +15,25 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  if (!century) {
+    const amountMan = people.filter(item => item.sex === 'm').length;
+
+    const sumOld = people.filter(item => item.sex === 'm')
+      .reduce((sum, person) =>
+        sum + (person.died - person.born), 0);
+
+    return Math.ceil((sumOld / amountMan * 100)) / 100;
```

Comment:
don't need 2 times filter men

```suggestion
    const men = people.filter(item => item.sex === 'm');

    const sumOld = men.reduce((sum, person) =>
        sum + (person.died - person.born), 0);

    return Math.ceil((sumOld / men.length * 100)) / 100;
```